pydantic/_internal/_typing_extra.py

```diff
@@ -128,89 +128,6 @@ def annotated_type(tp: Any, /) -> Any | None:
     return get_args(tp)[0] if is_annotated(tp) else None
 
 
-def unpack_annotated(annotation: Any, /) -> tuple[Any, list[Any]]:
```

Comment:
This logic is not lost, it is present in `typing-inspection` and type aliases unpacking is configurable there